### PR TITLE
Bump obt beta

### PR DESCRIPTION
**Pull request by notlee**
- Bump Build Tools beta
- Use node 15 in workflows

### CODE REVIEW

File: .github/workflows/publish-to-npm.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2.1.4
       with:
-        node-version: '12.x'
+        node-version: '15.x'
```
Comment (JakeChampion) [inline .github/workflows/publish-to-npm.yml]: Are we using node 15 as a way to get npm 7?

Comment (notlee) [reply .github/workflows/publish-to-npm.yml]: Yeah, I figured we'd probably do this for component workflows and should run ci-tools tests in the same environment -- it's actually not necessary for the publish step. Would you prefer we stick to an LTS release? Not sure there's a straightforward way to configure the npm version with `setup-node` but we could run an extra install command...

Comment (JakeChampion) [reply .github/workflows/publish-to-npm.yml]: What we would need to do is ensure that the test workflow uses the versions of NodeJS that we state are supported.

In the build service we use the version of the NodeJS that we support and then install the version of npm we support.

https://github.com/Financial-Times/origami-build-service/blob/master/.github/workflows/test.yml#L8-L11

Comment (notlee) [reply .github/workflows/publish-to-npm.yml]: I actually removed that commit. The only "test" we're running is prettier so there was no need. Will use node 12 + npm 7 with component workflows

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v2.2.1-beta.2 :tada:

**Comment by notlee** [comment]
this release was on the wrong major because the last commit on the base branch had two release tags:
`tag: v3.0.0-beta.4, tag: v2.2.1-beta.1`. I'm going to sort this out manually. We shouldn't find ourselves
in this situation again now `origami-version`can release major betas with the beta and major release labels
Screenshot 2021-01-29 at 11 32 39

**Pull request merged**